components/molecule/notification/src/index.js

```diff
@@ -217,6 +224,8 @@ MoleculeNotification.propTypes = {
 export {
   POSITION,
   POSITION as moleculeNotificationPosition,
+  ALIGN_ITEMS,
```

Comment:
remove this line plz. The CAPITAL_LETTERS naming is just 4 retrocompatibility. It will be removed in next major